dev-qt/qtimageformats/qtimageformats-5.12.3.ebuild

```diff
@@ -10,20 +10,18 @@ if [[ ${QT5_BUILD_TYPE} == release ]]; then
 	KEYWORDS="amd64 ~arm arm64 ~hppa ppc64 ~sparc x86"
 fi
 
-IUSE="jpeg2k mng"
+IUSE="mng"
 
 DEPEND="
 	~dev-qt/qtcore-${PV}
 	~dev-qt/qtgui-${PV}
 	media-libs/libwebp:=
 	media-libs/tiff:0
-	jpeg2k? ( media-libs/jasper:= )
 	mng? ( media-libs/libmng:= )
 "
 RDEPEND="${DEPEND}"
 
 src_configure() {
-	qt_use_compile_test jpeg2k jasper
```

Comment:
@Pesa how can we safely disable jasper here without triggering automagic?

Reply:
Copy what `qt_use_compile_test` is doing, without the conditional.

But if you're going to remove jasper, there won't be any risk of automagic, right?

Reply:
hence why I'm pondering to just leave it like it is, once jasper is gone